apparmor.d/groups/virt/k3s

```diff
@@ -0,0 +1,176 @@
+  capability sys_admin,
+  capability sys_resource,
+
+  ptrace peer=@{profile_name},
+  ptrace (read) peer=unconfined,
+
+  network inet    dgram,
+  network inet6   dgram,
+  network inet    stream,
+  network inet6   stream,
+  network netlink raw,
+
+  mount  /var/lib/kubelet/pods/@{uuid}/volumes/kubernetes.io~*/{,**/},
+  umount /var/lib/kubelet/pods/@{uuid}/volumes/kubernetes.io~*/{,**/},
+
+  signal (send, receive) set=term,
+
+  @{exec_path} mr,
+  /{usr/,}bin/kmod rPx,
+  /{usr/,}bin/mount rPx,
+  /{usr/,}bin/systemd-run rix,
+
+  # These are all symbolic links to xtables-nft-multi on Ubuntu 22.04
+  /{usr/,}{s,}bin/iptables rPx -> xtables-nft-multi,
+  /etc/alternatives/iptables rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/iptables-legacy rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/xtables-nft-multi rPx,
+
+  /{usr/,}{s,}bin/iptables-save rPx -> xtables-nft-multi,
+  /etc/alternatives/iptables-save rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/iptables-legacy-save rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/xtables-nft-multi rPx,
+
+  /{usr/,}{s,}bin/iptables-restore rPx -> xtables-nft-multi,
+  /etc/alternatives/iptables-restore rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/iptables-legacy-restore rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/xtables-nft-multi rPx,
+
+  /usr/libexec/kubernetes/kubelet-plugins/volume/exec/nodeagent~uds/uds rix,
+  /var/lib/rancher/k3s/data/[0-9a-f]*/bin/* rix,
+
+  /usr/libexec/kubernetes/kubelet-plugins/volume/exec/{,**} r,
+  /usr/share/mime/globs2 r,
+
+  /etc/machine-id            r,
+  /etc/rancher/k3s/{,**}     r,
+  /etc/rancher/k3s/k3s.yaml  rw,
+  /etc/rancher/node/password r,
+
+  /var/lib/rancher/k3s/{,**}        r,
+  /var/lib/rancher/k3s/agent/**     rw,
+  /var/lib/rancher/k3s/server/**    rw,
+  /var/lib/rancher/k3s/server/db/** rwk,
+
+  # k3s want's to basically manage all directories and create some specific files.
+  /var/lib/kubelet/{,**/} rw,
+  /var/lib/kubelet/{cpu_manager_state,memory_manager_state} r,
+  /var/lib/kubelet/device-plugins/{,DEPRECATION,kubelet.sock} rw,
+  /var/lib/kubelet/pod-resources/{kubelet.sock,[0-9]*} rw,
+  /var/lib/kubelet/pods/@{uuid}/containers/*/[0-9a-f]* rw,
+  /var/lib/kubelet/pods/@{uuid}/etc-hosts rw,
+  /var/lib/kubelet/pods/@{uuid}/plugins/kubernetes.io~*/{,**} rw,
+  /var/lib/kubelet/pods/@{uuid}/volumes/kubernetes.io~*/{,**} rw,
+  /var/lib/kubelet/pods/@{uuid}/**/ca.crt rw,
+  /var/lib/kubelet/pods/@{uuid}/**/namespace rw,
+  /var/lib/kubelet/pods/@{uuid}/**/token rw,
+
+  /var/log/containers/         r,
+  /var/log/containers/**       rw,
+  /var/log/rancher/{,**}       r,
+  /var/log/kubelet/{,**}       r,
+  /var/log/kubernetes/{,**}    r,
+  /var/log/kubernetes/audit/** rw,
+  /var/log/pods/{,**}          r,
+  /var/log/pods/{,**/}         rw,
+  /var/log/pods/**/[0-9]*.log  rw,
+
+  @{HOME}/.kube/cache/discovery/{,**}         rw,
+  @{HOME}/.kube/cache/http/[0-9a-z]*          rw,
+  @{HOME}/.kube/cache/http/.diskv-temp/[0-9]* rw,
```

Comment:
```suggestion
  owner @{HOME}/.kube/cache/discovery/{,**}         rw,
  owner @{HOME}/.kube/cache/http/[0-9a-z]*          rw,
  owner @{HOME}/.kube/cache/http/.diskv-temp/[0-9]* rw,
```
Usually you always have `owner @{HOME}` never `@{HOME}` (with some exception of course)